weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -104,12 +105,11 @@ export const useCallsForQuery = (
     undefined,
     sortBy,
```

Comment:
i think its this sort by here, we try to sort by outputdump, but dont have the requested column

Reply:
i think we can remove the sortby since we stitch the results together later anyway

Reply:
ah good call, that explains why its a very infrequent error.